strings/src/main/res/values-cs/strings.xml

```diff
@@ -1,997 +1,992 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <resources xmlns:tools="http://schemas.android.com/tools">
-  <!--Text for the topmost button on the main screen. It should be short and action-oriented.-->
```

Comment:
Could you keep the same indentation so that it's clear what strings were really changed?